docker-test.yml

```diff
@@ -38,6 +38,10 @@ services:
     environment:
       BOILERPLATE_ENV: test
       ALEGRE_PORT: 3100
+  presto:
+    environment:
+      BOILERPLATE_ENV: test
+      ALEGRE_PORT: 8000
```

Comment:
`ALEGRE_PORT`? 8000 is the Presto port, no?

Reply:
whoops! Fixed